signalfx/resource_signalfx_time_chart.go

```diff
@@ -1147,7 +1147,8 @@ var validateUnitTimeChart = validation.StringInSlice([]string{
 	"Byte",
 	"Kibibyte",
 	"Mebibyte",
-	"Gigibyte",
+	"Gibibyte",
+	"Gigibyte", // This is an historical typo that has been corrected on the server side frontend
```

Comment:
Should the old value be dropped to help existing users notice the change?

Reply:
An alternative could be to emit a warning if the value `Gigibyte` is passed: https://developer.hashicorp.com/terraform/plugin/sdkv2/schemas/schema-behaviors#validatefunc

Reply:
We're about to do a major release so I'm ok with just removing it and making it a breaking change that we'll roll into the major release.

Reply:
Done!